docs/_components/modals-basic.html

```diff
@@ -0,0 +1,25 @@
+---
+title: "Basic"
+description: |
```

Comment:
Hey @Jing-Ran 

I think you got some time to get the new changes in as we receive components from other contributors.

Did you manage to see the showcase website locally?